core/toolbox/toolbox.js

```diff
@@ -238,6 +238,68 @@ Blockly.Toolbox.prototype.attachEvents_ = function(container,
   this.boundEvents_.push(keyDownEvent);
 };
 
+/**
+ * Handles on click events for when the toolbox or toolbox items are clicked.
+ * @param {!Event} e Click event to handle.
+ * @protected
+ */
+Blockly.Toolbox.prototype.onClick_ = function(e) {
```

Comment:
Do protected functions normally end in underscores too? I'm genuinely asking because I haven't used protected very often. The style guide only mentions it being required for private, but afaict it doesn't say you can't for other visibilities so I'm not sure.

Reply:
In Blockly yes, but I am not entirely sure what the norm for this is or what the story is behind why blockly does this.